mypy/types.py

```diff
@@ -1490,7 +1496,7 @@ def visit_tuple_type(self, t: TupleType) -> str:
         s = self.list_str(t.items)
         if t.fallback and t.fallback.type:
             fallback_name = t.fallback.type.fullname()
-            if fallback_name != 'builtins.tuple':
+            if fallback_name not in ('builtins.tuple', 'builtins.object'):
```

Comment:
I intended this change to preserve existing message format that otherwise would have changed; but I can't produce any specific examples where it's necessary, so I can get rid of this.